etna/transforms/embeddings/models/ts2vec.py

```diff
@@ -267,11 +275,50 @@ def load(cls, path: pathlib.Path) -> "TS2VecEmbeddingModel":
         path:
             Path to load object from.
 
+            - if `path` is not None and `model_name` is None, load the local model from `path`.
+            - if `path` is None and `model_name` is not None, save the external `model_name` model to the etna folder in the home directory and load it.
+            If `path` exists, external model will not be loaded.
+            - if `path` is not  None and `model_name` is not None, save the external `model_name` model to `path` and load it.
+            If `path` exists, external model will not be loaded.
+
+        model_name:
+            name of external model to load. To get list of available models use `list_models` method.
+
         Returns
         -------
         :
             Loaded object.
+
+        Raises
+        ------
+        ValueError:
+            If non of parameters path and model_name are set.
+        NotImplementedError:
+            If model_name is not from list of available model names
         """
+        if model_name is not None:
+            if path is None:
+                path = _DOWNLOAD_PATH / f"{model_name}.zip"
+            if os.path.exists(path):
+                warnings.warn(
+                    f"Path {path} already exists. Model {model_name} will not be downloaded. Loading existing local model."
+                )
+            else:
+                directory = os.path.dirname(path)
+                # If path not in current directory and it doesn't exist
+                if directory and not os.path.exists(directory):
+                    os.makedirs(directory)
+
+                try:
+                    url = f"http://etna-github-prod.cdn-tinkoff.ru/embeddings/ts2vec/{model_name}.zip"
+                    request.urlretrieve(url=url, filename=path)
+                except HTTPError:
+                    raise NotImplementedError(
```

Comment:
Why is it `NotImplementedError`, not `ValueError`?

Reply:
Like [here](https://github.com/etna-team/etna/blob/e5cc8d68996e8fd22c67c0baa1c404ae81911435/etna/datasets/internal_datasets.py#L150)